cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/distributed/AbstractDistributedProgramRunner.java

```diff
@@ -295,9 +295,17 @@ private URI getLogBackURI(Program program, File programDir, File tempDir) throws
 
   /**
    * Sub-class overrides this method to launch the twill application.
+   *
+   * @param program the program to launch
+   * @param options the options for the program
+   * @param localizeResources a mutable map for adding extra resources to localize
+   * @param tempDir a temporary directory for this launch. Sub-class can use it to create resources for localization
```

Comment:
Sub-classes can use it to create resources for localization
which require cleanup after launching is completed.